### ISSUE TITLE
Missing weights for inference on missing sequences

### ISSUE THREAD

**Issue by brunotakara** [BrainLesion/tutorials#39]
Hello again,

I am trying to make inferences on missing sequences combinations, but I am getting this following error:

```
 No weights found for model fla-o and selection best. 
Available models: ['t1-t1c-t2-fla', 't1-t1c-fla', 't1-t1c', 't1c-fla', 't1c-o', 'fla-o', 't1-o']
```

So far I was able to do it without the t2: 't1-t1c-fla' and without both t2 and flair sequence: 't1-t1c'. All other combinations raise this error, the only thing I am doing different than the others is commenting the arguments t1, t1c, fla and t2 within the inferer.infer(), is there something I am missing? Thanks!

**Comment by neuronflow** [comment]
Thanks for your interest in our tutorials :)

It sounds like you are using the AURORA package?

Which operating system?

Which version of the package?

Which operating system?

This sounds more like an issue with the package rather than an issue with the tutorial. So probably the issue should be [here](https://github.com/BrainLesion/AURORA/issues)

**Comment by brunotakara** [comment]
Yes, it is the AURORA package, I am running it on google colab, my operating system is windows but I think in colab it uses ubuntu

It says after installation: 
`Successfully installed brainles_aurora-0.2.3 ... and a list with all other packages `

Then I cloned the repo: `!git clone https://github.com/BrainLesion/tutorials.git`

And ran these imports:
`from brainles_aurora.inferer import AuroraInferer, AuroraInfererConfig`

Then setting config as follows:

```
config = AuroraInfererConfig(
    tta=True,
    sliding_window_batch_size=8,
    cuda_devices="0"
)

inferer = AuroraInferer(config=config)
```

**Comment by evamariie** [comment]
I tried to reproduce this ([colab link](https://colab.research.google.com/github/BrainLesion/tutorials/blob/main/AURORA/tutorial.ipynb) to the AURORA tutorial notebook)  with and without the parameters mentioned above. It worked.
Are you running the tutorials notebook or did you install the Brainles_Aurora and use it separately?

We also had this error, but it was due to an outdated version of Aurora.
Since colab re-installs it every time, it may also be related to the "tutorial" folder in Google Drive. An option would be to delete it?
